downstream/titles/release-notes/async/aap-25-4-20-nov.adoc

```diff
@@ -0,0 +1,74 @@
+[[aap-25-4-20-nov]]
+
+= {PlatformNameShort} patch release November 20, 2024
+
+The following enhancements and bug fixes have been implemented in this release of {PlatformNameShort}.
+
+== Enhancements
+
+* With this release, a redirect page has now been implemented that will be exhibited when you navigate to the root ``/`` for each component's stand-alone URL. The API endpoint remains functional. This affects {EDAName}, {ControllerName}, {OperatorPlatformNameShort}, and {OCPShort}.
+
+
+== Bug fixes
+
+=== General
+
+With this update, the following CVEs have been addressed:
+
+link:https://access.redhat.com/security/cve/cve-2024-9902[CVE-2024-9902] ansible-core: Ansible-core user may read/write unauthorized content.
+
+link:https://access.redhat.com/security/cve/cve-2024-8775[CVE-2024-8775] ansible-core: Exposure of sensitive Information in Ansible vault files due to Improper logging.
+
+
+=== {PlatformNameShort}
+
+* Fixed an issue where the user is unable to filter out hosts on inventory groups where it returns a *Failed to load* options on {PlatformNameShort} UI.
+
+=== Execution Environment
+
+* Update *pywinrm* to 0.4.3 in *ee-minimal* and *ee-supported* container images to fix Python 3.11 compatibility.
+
+=== {OperatorPlatformNameShort}
+
+* Fixed a syntax error when `bundle_cacert_secret` is defined due to incorrect indentation.
+
+* Fixed an issue where the default operator catalog for {PlatformNameShort} aligns to cluster-scoped versus namespace-scoped.
+
+* Added the ability to set tolerations and `node_selector` for the Redis *statefulset* and the gateway deployment.
+
+* Ensure the platform URL status is set when *Ingress* is used to resolve an issue with {Azure} on Cloud managed deployments, where the {PlatformNameShort} operator fails to finish because it is looking for {OCPShort} routes that are not available on Azure Kubernetes Service.
```

Comment:
It's too long a sentence; consider breaking it down for clarity.

Reply:
Done.